opensrp-chw-core/src/test/java/org/smartregister/chw/core/activity/CoreAncRegisterActivityTest.java

```diff
@@ -66,4 +97,42 @@ public void testGetFamilyBaseEntityId() {
         Assert.assertEquals(activity.getFamilyName(), "familyName");
         Assert.assertEquals(activity.getPhoneNumber(), "phone_number");
     }
+
+    @Test
+    public void testOnResumption() {
+        NavigationMenu menu = Mockito.mock(NavigationMenu.class);
+        NavigationAdapter adapter = Mockito.mock(NavigationAdapter.class);
+
+        Mockito.doReturn(adapter).when(menu).getNavigationAdapter();
+        ReflectionHelpers.setStaticField(NavigationMenu.class, "instance", menu);
+
+        ReflectionHelpers.setField(activity, "presenter", presenter);
+        activity.onResumption();
+
+        Mockito.verify(adapter).setSelectedView(Mockito.anyString());
+    }
+
+    @Test
+    public void testOnBackPressed() {
+        CoreAncRegisterActivity spyActivity = Mockito.spy(activity);
+        spyActivity.onBackPressed();
+        Mockito.verify(spyActivity).onBackPressed();
+    }
+
+    @Test
+    public void testOnCreateOptionsMenu() {
+        Assert.assertTrue(activity.onCreateOptionsMenu(menu));
+    }
+
+    @Test
+    public void testStartMe() {
```

Comment:
Kindly follow the test naming convention `test[MethodName][ExpectedResult][Parameters]`